tools/peptideshaker/searchgui.xml

```diff
@@ -330,19 +887,25 @@
             <param name="variable_modifications" value="oxidation of m"/>
             <param name="min_charge" value="1"/>
             <param name="max_charge" value="3"/>
-            <param name="advanced_type_selector" value="advanced"/>
-            <param name="xtandem_selector" value="yes"/>
-            <param name="xtandem_selector.xtandem_refine_selector" value="yes"/>
-            <param name="omssa_selector" value="no"/>
-            <param name="msgf_selector" value="yes"/>
-            <param name="ms_amanda_selector" value="no"/>
+            <param name="XTANDEM_selector" value="True"/>
+            <param name="xtandem.xtandem_advanced" value="yes"/>
+            <param name="xtandem_advanced.xtandem_refine_selector" value="yes"/>
+            <param name="OMSSA_selector" value="False"/>
+            <param name="MSGF_selector" value="True"/>
+            <param name="TIDE_selector" value="False"/>
+            <param name="MYRIMATCH_selector" value="False"/>
```

Comment:
Mh. `plenemo test searchgui.xml` works for me. At least after updating the test data.
peptideshaker on the new input of searchgui gives me this error:

```
  Picked up JAVA_TOOL_OPTIONS: -javaagent:/usr/share/java/jayatanaag.jar
|  java.lang.NullPointerException
|  at com.compomics.util.experiment.io.identifications.idfilereaders.MzIdentMLIdfileReader.parseFixedPtms(MzIdentMLIdfileReader.java:832)
|  at com.compomics.util.experiment.io.identifications.idfilereaders.MzIdentMLIdfileReader.parseFile(MzIdentMLIdfileReader.java:593)
|  at com.compomics.util.experiment.io.identifications.idfilereaders.MzIdentMLIdfileReader.getAllSpectrumMatches(MzIdentMLIdfileReader.java:244)
|  at eu.isas.peptideshaker.fileimport.FileImporter$IdProcessorFromFile.importPsms(FileImporter.java:660)
|  at eu.isas.peptideshaker.fileimport.FileImporter$IdProcessorFromFile.importFiles(FileImporter.java:450)
|  at eu.isas.peptideshaker.fileimport.FileImporter.importFiles(FileImporter.java:153)
|  at eu.isas.peptideshaker.PeptideShaker.importFiles(PeptideShaker.java:225)
|  at eu.isas.peptideshaker.cmd.PeptideShakerCLI.createProject(PeptideShakerCLI.java:708)
|  at eu.isas.peptideshaker.cmd.PeptideShakerCLI.call(PeptideShakerCLI.java:177)
|  at eu.isas.peptideshaker.cmd.PeptideShakerCLI.main(PeptideShakerCLI.java:1025)
|  Picked up JAVA_TOOL_OPTIONS: -javaagent:/usr/share/java/jayatanaag.jar
|  Reindexing: input_database.fasta. (changes in the file detected)
|  Reindexing: searchgui_tinyspectra1.mgf. (changes in the file detected)
| Dataset Job Standard Error:
```
Have you seen this before?

What is your error with planemo?